apps/events.md

```diff
@@ -127,105 +150,448 @@ cart: {
-coupon: null
-couponCampaign: null
-customerEmail: null
-grandAmount: "76,01 €"
-grandAmountNote: "components.productComponent.priceExclusiveVat"
-grandTotal: {amount: 76.01, currency: "EUR", formatted: "€76,01"}
-minimumOrderValue: null
-mustAcceptTermsAndConditions: false
-netAmount: "76,01 €"
-paymentLineItem: {lineItemPrice: {…}, paymentMethod: {…}}
-pickupToken: "ZjM3M2Q2YmY4jkFjYWRlZTIzZTBlYzQwMDU4MjYzZjYwNDNhZGY0NWM1N2JiNjZhMGI0YWNlNWFkYzU4ZTQ3OF8xNTQzOTM5MjIx"
-productLineItems: [{…}, {…}, {…}]
-registerSessionUrl: "https://pm.epages.com/epages/apidocu.sf/?ObjectPath=/Shops/apidocu/AnonymousUsers/2/Baskets/46072&ChangeAction=PickupBasket&PickupToken=NTRjZWJmNjdhNzNlYTUzNDAxZTgyZTc4ODYwYTliMDUxMDIzNDQ2OWY2NWQ5NWRmN2Q2YmVjZjVjNzljOTQ2N18xNDgyMjI3OTE3"
-shippingAddress: null
-shippingLineItem: {lineItemPrice: {…}, shippingMethod: {…}}
-status: null
-subAmount: "76,01 €"
-taxType: "NET"
-taxes: []
-totalNumberOfItems: 3
-_links: null
+cart: {
+  billingAddress: null
+  canHaveCoupon: true
+  cartId: "5C06901F-150C-3D9B-B144-D509AB34875"
+  checkoutButtons: [{…}]
+  checkoutState: null
+  checkoutUrl: "https://pm.epages.com/epages/apidocu.sf/?ObjectPath=/Shops/apidocu/AnonymousUsers/1/Baskets/46072&ChangeAction=PickupBasket&PickupToken=MjNiZThlOGM0MjAzZWQ0N2ZjYmZmZDFiZjI3OTQxMzkwOWY4ZjZlOGE4NTM0ZWIxMjg3NmY2NzBiY2IxYzQ1OV8xNDgyMjIzMDcz"
+  coupon: null
+  couponCampaign: null
+  customerEmail: null
+  grandAmount: "76,01 €"
+  grandAmountNote: "components.productComponent.priceExclusiveVat"
+  grandTotal: {amount: 76.01, currency: "EUR", formatted: "€76,01"}
+  minimumOrderValue: null
+  mustAcceptTermsAndConditions: false
+  netAmount: "76,01 €"
+  paymentLineItem: {lineItemPrice: {…}, paymentMethod: {…}}
+  pickupToken: "ZjM3M2Q2YmY4jkFjYWRlZTIzZTBlYzQwMDU4MjYzZjYwNDNhZGY0NWM1N2JiNjZhMGI0YWNlNWFkYzU4ZTQ3OF8xNTQzOTM5MjIx"
+  potentialBasketDiscounts: null
+  productLineItems: [{…}, {…}, {…}]
+  registerSessionUrl: "https://pm.epages.com/epages/apidocu.sf/?ObjectPath=/Shops/apidocu/AnonymousUsers/2/Baskets/46072&ChangeAction=PickupBasket&PickupToken=NTRjZWJmNjdhNzNlYTUzNDAxZTgyZTc4ODYwYTliMDUxMDIzNDQ2OWY2NWQ5NWRmN2Q2YmVjZjVjNzljOTQ2N18xNDgyMjI3OTE3"
+  shippingAddress: null
+  shippingLineItem: {lineItemPrice: {…}, shippingMethod: {…}}
+  status: null
+  subAmount: "76,01 €"
+  taxType: "NET"
+  taxes: []
+  totalBasketDiscount: null
+  totalNumberOfItems: 3
+  _links: null
+}
 ```
 
-## Cart:add event
+### cart:add
 
-To make use of this event, you need to add the following snippet to your code:
+To get more information about this event, you can use the following snippet:
 
 ```js
 if (window.eComEventTarget) {
   window.eComEventTarget.addEventListener('cart:add', function (event) {
-    console.log('new cart data:',event.detail.cart,'added product:',event.detail.product,'quantity:',event.detail.quantity)
+    console.log('cart:add', {
+      new_cart_data: event.detail.cart,
+      added_product: event.detail.product,
```

Comment:
Optional: These are descriptive names. However, given all the other examples I‘d prefer to name the variables the same as the property on `event.detail` for consistency. E.g. for `event.detail.cart`, it is "cart".

```suggestion
      cart: event.detail.cart,
      product: event.detail.product,
```

Or if you keep the naming, I suggest to make them camel case, which is more common in JavaScript. E.g. `newCartData`.